reports/project-groupby-status.ts

```diff
@@ -114,7 +114,7 @@ function getBreakdown(
   drillIn(drillInName(name, 'yellow'), `${name} with a status yellow`, groupByData.flagged.yellow)
 
   groupByData.flagged.inProgressDuration = issues.filter(issue => {
-    if (issue.project_in_progress_at) {
+    if (issue.project_stage === ProjectStages.InProgress) {
```

Comment:
even completed items will have the `project_in_progress_at` time set.  Should have filtered for the current state is in progress.